lib/shelley/test/unit/Cardano/Wallet/Shelley/TransactionSpec.hs

```diff
@@ -162,23 +183,25 @@ spec = do
         res `shouldBe` Right (FeeEstimation 165413 165413)
 
 estimateMaxInputsTests
-    :: Cardano.NetworkId
+    :: forall k. (TxWitnessTagFor k, Typeable k)
+    => Cardano.NetworkId
+    -> [(Word8, Word8)]
```

Comment:
Doc comment? 🙏 

I think a name like _estimateMaxInputs**GivenOutputs**Tests_ would make the call-site tuples more easy to understand too.

Reply:
> I think a name like estimateMaxInputsGivenOutputsTests would make the call-site tuples more easy to understand too.

Ok, I see `estimateMaxInputsTests` is not _only_ goldens. Then I'm less sure...

Reply:
> estimateMaxInputsGivenOutputsTests

Estimating max inputs is **always** given some outputs though.

Reply:
Yes, but I think these tests should be easily understood without already clearly knowing that. 

Adding a doc comment to `[(Word8, Word8)]` definitely helps.

But my idea with `estimateMaxInputsGivenOutputsTests` was to hint the same, already at the _call-site_, similar to  the impossible `[(outputs: 1, maxInputs: 2)]`, or the perhaps overkill `EstimateMaxInputsGolden { outputs, maxInputs :: Word8  }`.